src/ancillary/__tests__/anchor-request-params-parser.test.ts

```diff
@@ -97,4 +119,18 @@ describe('AnchoRequestParamsParser', () => {
     expect(params.timestamp).toEqual(new Date(TIMESTAMP_ISO))
     expect((params as RequestAnchorParamsV2).genesisFields).toEqual(CAR_FILE_FAKE_GENESIS_FIELDS)
   })
+
+  test('parses CAR file with signed genesis with capCID properly', () => {
+    const validation = parser.parse(CAR_FILE_REQUEST_EXAMPLE_SIGNED_GENESIS_WITH_CAPCID as ExpReq)
+    expect(isRight(validation)).toEqual(true)
+
+    const params: RequestAnchorParams = (validation as Right<RequestAnchorParams>).right
+    expect(params.capCID).toEqual(FAKE_CAPCID)
+  })
+
+  test('throws error on invalid car file', () => {
```

Comment:
this does not throw an error.  the validator returns the junk

```
Here is invalid validation: {"_tag":"Left","left":[{"trail":[{"key":"","type":{"name":"RequestAnchorParamsV2"},"actual":{"0":67,"1":114,"2":101,"3":97,"4":116,"5":101,"6":100,"7":66,"8":121,"9":67,"10":104,"11":97,"12":116,"13":71,"14":80,"15":84,"16":52,"17":89,"18":111,"19":117,"20":99,"21":97,"22":110,"23":85,"24":115,"25":101,"26":84,"27":104,"28":105,"29":115,"30":83,"31":116,"32":114,"33":105,"34":110,"35":103}}]}]}
```

@oed @ukstv shouldn't the validator throw an error on this junk?

Reply:
i can separate out this part into a separate pr since it is not a change affected by the capCID